master/static/srv/get_experiment_groups.sql

```diff
@@ -0,0 +1,3 @@
+SELECT id, name, project_id, (SELECT COUNT(*) FROM experiments WHERE group_id = g.id) AS num_experiments
```

Comment:
I don't know for sure, if this might be converted to bun or if this is provably more efficient.
In `get_workspaces.sql` we count experiments for each project in one table scan / query:

```sql
WITH
[....]
exp_count_by_project AS (
  SELECT COUNT(*) AS count, project_id FROM experiments
  GROUP BY project_id
)
```

I think this strategy would be better to do the counts first, and then in the final SELECT, you can JOIN or query this table of counts